client/components/keyed-suggestions/style.scss

```diff
@@ -1,6 +1,10 @@
-/*
+/**
+ * /*
```

Comment:
This comment doesn't look right, probably due to auto formatting

Reply:
Yeah, looks like `prettier` being creative!